### PR TITLE
Terminate unused sessions

### LINKED ISSUE #76: Disposal of abandoned sessions
When session is not active for more than defined timeout (configurable), the allocated compute resource should be disposed. We could consider complete disposal or hybernation, however we need to see what is the effect on license use in case of sessions with TeamForCapella projects

### PR DESCRIPTION
**Pull request by amolenaar**
# Description

Terminate idle sessions from the cluster.

It took a bit longer to get right. Created #434, which runs the cleanup job through a kubernetes cron job.

It seems to me this is the most appropriate solution at the moment: a simple timer and a cleanup job.

Resolves #76 

# Testing

Live tested.

# Checklist

- [x] My code follows the guidelines of this project: [CONTRIBUTING.md](/CONTRIBUTING.md)
- [x] I updated the documentation with the new functionality
- [x] I went through the code and removed all print statements, breakpoints and unused code
- [x] Error handling for all possible scenarios has been implemented
- [x] I've add logging for all relevant operations
- [x] I added the appropriate labels to this PR (enhancement/bug/documentation, effort, priority)

### CODE REVIEW

**Review by dominik003 — COMMENTED** [review]
Looks good! There is only a small problem with the function name that needs to be fixed before merging.

File: backend/capellacollab/__main__.py [hunk 1]
```diff
@@ -41,6 +43,11 @@ async def startup_event():
     migration.migrate_db(engine)
 
 
+@app.on_event("startup")
+async def startup_event():
+    await terminate_idle_sessions_in_background()
```
[the remark this answers is not in the document]
Comment (amolenaar) [reply backend/capellacollab/__main__.py:48]: Fair point

**Review by dominik003 — APPROVED** [review]

**Comment by MoritzWeber0** [comment]
Nice PR, thank you @amolenaar. Follow up issue: https://github.com/DSD-DBS/capella-collab-manager/issues/456

**Review by MoritzWeber0 — APPROVED** [review]
Nice PR, thank you @amolenaar. Follow up issue: https://github.com/DSD-DBS/capella-collab-manager/issues/456

**Pull request merged**